dist/jsonapi-client-test.js

```diff
@@ -435,7 +444,12 @@ Transport.prototype._action = function(method, url, data, callback) {
         response = JSON.parse(err.response.text);
       } catch(e) {
         console.error("Transport Error: " + JSON.stringify(err));
-        return callback(new Error("Invalid response from server"));
+        return callback(Transport._defaultError(response));
+      }
+
+      if (!(response.errors instanceof Array)) {
```

Comment:
I didn't even thing about `Array.isArray` vs `instanceof Array` until now, so I'm not sure. I'll leave it to your better judgement. :)